### DISCUSSION TITLE
COPR builds failing

### DISCUSSION THREAD

**Discussion by FluxZA** [wezterm/wezterm#4981] [Q&A] [answered]
### What Operating System(s) are you running on?

Linux Wayland

### Which Wayland compositor or X11 Window manager(s) are you using?

Mutter

### WezTerm version

20240128-202157-1e552d76

### Ask your question!

I went to see which versions were available on the Copr repo and I noticed the builds have failed. I assume I should wait for them to succeed before trying to install from the repo?

I would be interested in helping resolve the Copr issue if that is an area you accept contributions in.

**Comment by wez** [comment, marked as answer]
Thanks for reporting this. It looks from https://download.copr.fedorainfracloud.org/results/wezfurlong/wezterm-nightly/opensuse-tumbleweed-aarch64/06995683-wezterm-nightly/builder-live.log.gz as though this is some kind of weird suse specific burp. I don't think there is anything to actively do about this: the suse distros are generally flakey in this way and eventually fix themselves

**Reply by FluxZA** [reply]
Okay great, thanks for the info.

Out of interest do you plan to add the stable versions to the Copr at some point?
